github/authorizations.go

```diff
@@ -302,10 +306,14 @@ func (s *AuthorizationsService) Check(ctx context.Context, clientID string, toke
 // The returned Authorization.User field will be populated.
 //
 // GitHub API docs: https://developer.github.com/v3/oauth_authorizations/#reset-an-authorization
-func (s *AuthorizationsService) Reset(ctx context.Context, clientID string, token string) (*Authorization, *Response, error) {
-	u := fmt.Sprintf("applications/%v/tokens/%v", clientID, token)
+func (s *AuthorizationsService) Reset(ctx context.Context, clientID string, token string, accessToken string) (*Authorization, *Response, error) {
```

Comment:
s/clientID string, token string, accessToken string/clientID, token, accessToken string/

Please also add a comment about the difference between `token` and `accessToken` and how to know which is for what purpose.